docs/ref/request-response.txt

```diff
@@ -331,9 +331,15 @@ Methods
 .. method:: HttpRequest.get_port()
 
     Returns the originating port of the request using information from the
+    ``HTTP_X_FORWARDED_HOST`` (if :setting:`USE_X_FORWARDED_HOST` is enabled),
     ``HTTP_X_FORWARDED_PORT`` (if :setting:`USE_X_FORWARDED_PORT` is enabled)
     and ``SERVER_PORT`` ``META`` variables, in that order.
 
+ .. versionchanged:: 3.0.5
```

Comment:
Minor typo; leading space causes incosistent styling in documentation
```suggestion
.. versionchanged:: 3.0.5
```

Reply:
fixed.